app/javascript/components/request-workflow-status/data.js

```diff
@@ -89,19 +89,25 @@ export const workflowStatusData = (response) => {
     return undefined;
   }
   const rows = response.context ? rowData(response.context) : [];
+  let lastId = '';
+  if (rows.length > 0) {
+    lastId = rows[rows.length - 1].id;
+  }
   if (response.context && response.context.State) {
```

Comment:
So basically, something like the following (not sure this works, but you get the idea)

```suggestion
  if (response.context && response.context.State && !response.context.State.FinishedTime) {) {
```

Reply:
and then of course you don't need any of the other changes.

Reply:
Sounds good, made the fix and repushed